src/goto-symex/goto_trace.h

```diff
@@ -9,7 +9,7 @@
 #include <irep2/irep2.h>
 #include <util/migrate.h>
 #include <vector>
-
+#include <string_view>
```

Comment:
missing new line

Reply:
Newline has been added after the include.